city_scrapers/spiders/example.py

```diff
@@ -0,0 +1,231 @@
+import re
+from collections import defaultdict
+from datetime import datetime
+
+import scrapy
+from city_scrapers_core.constants import BOARD, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class StlAldermenSpider(CityScrapersSpider):
+    name = "stl_public_service"
+    agency = "St. Louis Board of Public Service"
+    timezone = "America/Chicago"
+    custom_settings = {"ROBOTSTXT_OBEY": False}
+    start_urls = [
+        (
+            "https://www.stlouis-mo.gov/government/departments/public-service/index.cfm"
```

Comment:
We want the `start_urls` link to be the link to where the meeting materials are posted. I believe the website we want is https://www.stlouis-mo.gov/government/departments/public-service/documents/meeting-materials.cfm.